kitchen.yml

```diff
@@ -0,0 +1,261 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+# For help on this file's format, see https://kitchen.ci/
+driver:
+  name: docker
+  use_sudo: false
+  privileged: true
+  run_command: /lib/systemd/systemd
+  run_options:
+    add-host:
+      - 'localhost.localdomain:127.0.0.1'
```

Comment:
what is this option for? _(didn't find any info on kitchen doc so if you have any link, I'm interested also)_

Reply:
ok, `run_options` just pass options to docker when starting the container

Reply:
@daks [Extra command-line options to pass to `docker run` when starting the container.](https://github.com/test-kitchen/kitchen-docker#run_options).